api/checks/azure.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package checks
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"regexp"
+
+	"github.com/google/go-github/github"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// https://docs.microsoft.com/en-us/rest/api/azure/devops/build/artifacts/get?view=azure-devops-rest-4.1
+
+// BuildArtifacts is a wrapper for multiple BuildArtifact results.
+type BuildArtifacts struct {
+	Count int64           `json:"count"`
+	Value []BuildArtifact `json:"value"`
+}
+
+// BuildArtifact is an artifact published by a build.
+type BuildArtifact struct {
+	ID       int64            `json:"id"`
+	Name     string           `json:"name"`
+	Resource ArtifactResource `json:"resource"`
+}
+
+// GetReportURL produces a URL for fetching the report file, if applicable.
+func (b BuildArtifact) GetReportURL(organization string) (*url.URL, error) {
+	containerID := b.Resource.GetContainerID()
+	if containerID == "" {
+		return nil, errors.New("Failed to extract container ID")
+	}
+	result, err := url.Parse(fmt.Sprintf(
+		"https://dev.azure.com/%s/_apis/resources/Containers/%v",
```

Comment:
Link to documentation about "file containers"

Reply:
I'd love to. It's undocumented. I found it using the UI. Clicking the ... reveals `Copy to clipboard`

![image](https://user-images.githubusercontent.com/777956/49609164-ba298b80-f968-11e8-95f0-ca1e3b7b31a1.png)